app/routes/labs.missions/route.tsx

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { GridContainer } from '@trussworks/react-uswds'
+
+import fermiHeroImage from './Fermi_Earth_GWs.jpg'
+import { Meatball } from '~/components/meatball/Meatball'
+import type { BreadcrumbHandle } from '~/root/Title'
+
+import fleet from './AstroFleetChart.png'
+
+export const handle: BreadcrumbHandle = {
+  breadcrumb: 'Missions',
+}
+
+export default function () {
+  return (
+    <>
+      <img
+        alt="Fermi satellite with a GRB in the background, both positioned over earth"
+        src={fermiHeroImage}
+        height="600"
+        width="1800"
+        className="mobile:display-block tablet:display-block height-auto"
```

Comment:
Why not just `display-block`?

Reply:
because the image distorts

Reply:
<img width="484" alt="Screenshot 2024-01-03 at 10 53 35 AM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/d25b424a-1b47-4959-807a-f377f12adb24">
without hight-auto

Reply:
OK. How about `width-full height-auto`?